.github/workflows/auto-merge-dependabot-prs.yml

```
@@ -1,6 +1,9 @@
 name: Auto merge Dependabot PRs
 
-on: pull_request
+on:
+  pull_request:
+    paths:
+      - 'grafana/**'
```

was the intention to auto-merge dependabot PRs if they are *not* grafana? in that case this should be inverted, like so?

```suggestion
on:
  pull_request:
    paths-ignore:
      - 'grafana/**'
```

---

Oh dear, I think it's Friday afternoon :woman_facepalming: . Thanks for that catch, I've updated the commit now.